ProjectFiles/Assets/Scripts/Game/Game.cs

```diff
@@ -50,6 +51,12 @@ private void Awake()
             var commandLineArgs = new List<string>(Environment.GetCommandLineArgs());
             IsHeadless = commandLineArgs.Contains("-batchmode");
 
+            if (gameLoop == ServerClientSetting.Standalone)
+            {
+                RequestGameLoop(typeof(ClientGameLoop), new []{"standalone"});
```

Comment:
can be done later, but I really feel we should move away from passing string args to each gameloop in the future.

Reply:
Yeah we should 😂